pkg/commands/logtail/root.go

```diff
@@ -101,31 +101,39 @@ func (c *RootCommand) Exec(_ io.Reader, out io.Writer) error {
 		return err
 	}
 
+	failure := make(chan error)
 	sigs := make(chan os.Signal, 2)
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
 
 	// Start the output loop.
 	go c.outputLoop(out)
 
 	// Start tailing the logs.
-	go c.tail(out)
-
-	<-sigs
-	close(c.dieCh)
+	go c.tail(out, failure)
```

Comment:
An alternative to passing `failure` into `tail` would be something like
```
go func() {
    failure <- c.tail(out)
}()
```
I bring it up in case you think it'd simplify `c.tail`.

Reply:
Done